mc/Voyage-Mongo-Core.package/VOTMongoOperation.trait/instance/readDatabaseWith..st

```diff
@@ -0,0 +1,3 @@
+accessing
+readDatabaseWith: aBlock
+	^ self repository readingPool withDatabase: aBlock
```

Comment:
I don't understand why we have readingPool here. Wouldn't it be better to delegate just the readDatabaseWith: call to the repository and the repository decides?

Reply:
Furthermore I do not understand is it just that we keep readPool here that makes it mongo specific? Otherwise it shouldn't be called VOTMongoOperation

Reply:
I like the idea of just telling the repository `readDatabaseDo:` and `writeDatabaseDo:`instead of accessing the pools. And remove the trait. I'll create new commit with that.

Reply:
Pushed the new version.